tests/integration/authentication.spec.ts

```diff
@@ -31,11 +31,14 @@ describe(`Authentication integration tests`, () => {
     expect(secretKey.split(' ').length).toEqual(24);
   });
 
-  it('should be able to login from authentication page', async () => {
+  it('should be able to login from authentication page then logout', async () => {
     await wallet.clickSignIn();
     await wallet.loginWithPreviousSecretKey(SECRET_KEY);
     await wallet.waitForHomePage();
     const secretKey = await wallet.getSecretKey();
     expect(secretKey).toEqual(SECRET_KEY);
+    await wallet.clickSettingsButton();
+    await wallet.page.click('text=Sign Out');
+    await wallet.waitForLoginPage();
```

Comment:
Perhaps it's better to use an `expect` clause? I understand this may fail the tests if it doesn't show up, but nonetheless an explicit expect might be clearer. 

Could you select it with `page.$()` and expect for it to be defined?

Reply:
Sure ! this will be better indeed.